merkle/memory_merkle_tree_test.go

```diff
@@ -350,7 +357,10 @@ func TestBuildTreeBuildTwoChunks(t *testing.T) {
 
 	// Add to the tree, checking after each leaf
 	for l := 0; l < 8; l++ {
-		mt.AddLeaf(decodeHexStringOrPanic(leafInputs[l]))
+		_, _, err := mt.AddLeaf(decodeHexStringOrPanic(leafInputs[l]))
```

Comment:
nit: inline err in the if

Ditto for others.

Reply:
done